estimage/simpledata.py

```
@@ -35,24 +35,48 @@ def CONFIG_FILENAME(cls):
 
-class ProjCardIO(IniInDirMixin, ini.IniCardIO):
-    CONFIG_BASENAME = "projective.ini"
-    WHAT_IS_THIS = "projective card"
+    def _get_filepath(self, category):
+        if not datadir:
+            datadir = pathlib.Path(".")
+        return datadir / ret.stem_to_filename(category)
+
+    @staticmethod
+    def _set_file_path(io, ):
+        io.LOAD_FILENAME = path
```

Ruff (0.13.1)

Undefined name `path`

(F821)